src/effects/effects_base/internal/effectsrepositoryhelper.cpp

```
@@ -24,7 +24,7 @@ constexpr EffectFamily toEffectFamily(muse::audio::AudioResourceType type)
     case muse::audio::AudioResourceType::NyquistPlugin:
         return EffectFamily::Nyquist;
     case muse::audio::AudioResourceType::FluidSoundfont:
-    case muse::audio::AudioResourceType::MusePlugin:
+    case muse::audio::AudioResourceType::NativeEffect:
     case muse::audio::AudioResourceType::MuseSamplerSoundPack:
```

Cppcheck (2.20.0)

[style] The function 'wxFromStdString' is never used.

(unusedFunction)